requirements.txt

```diff
@@ -0,0 +1,2 @@
+pyota
+six
```

Comment:
[`six`](http://pythonhosted.org/six/) is a library that makes it easy to write code that "just works" in Python 2 and Python 3.